src/woo_search/search_index/tests/test_search_api.py

```diff
@@ -413,6 +413,152 @@ def test_query_default_search_uses_AND_instead_of_OR(self):
             data["results"][0]["record"]["uuid"], "da45268a-ab21-4a81-bfc4-b0430edf339b"
         )
 
+    def test_broken_query_string_syntax(self):
+        # broken quotes, AND not followed by operand
+        response = self.client.post(self.url, {"query": '"document two AND'})
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_query_with_exact_match_using_double_quotes(self):
+        index_document(
+            **IndexDocumentFactory.build(
+                uuid="d6eacab4-cb9f-42f7-abdf-719b358da923",
+                identifier="Document one",
+                omschrijving="The first document attached to this",
+            )
+        )
+        index_document(
+            **IndexDocumentFactory.build(
+                uuid="a8fce14e-88d1-4f60-a69b-bbcc7033afe9",
+                identifier="Document two",
+                omschrijving="Document two, the document that came after one",
+            )
+        )
+
+        with self.subTest("Exact search using double quotes"):
+            response = self.client.post(self.url, {"query": '"document one"'})
```

Comment:
I messed up when writing this test. This will test the `identifier` field which will be an exact match regardless of the quotes. We should change the `omschijving` and substring we want to test against.

Reply:
But it's right? The amount of results matches the expectation, and the subtest below asserts that not using quotes yields both results instead of the expected exact phrase match.

Reply:
This doesn't differentiate between partial matching and full matching. I think it would be more safe to change it up. Especially if something might change in the future.

I think using something like this as the description will more accurately test what the feature needs to be. 
"some text about **document one** that we will use for testen"

Because this was a problem that I had when trying to find a solution (because I missed that simple_query_string does all of this for us) while testing directly against ES.

Reply:
okay! I'll adapt it